db/migrations/0012.add_mivideo_tables.py

```diff
@@ -0,0 +1,16 @@
+from yoyo import step
+
+__depends__ = {'0009.add_meta_tables.py'}
```

Comment:
Please see my other migration-related comments. I don't really think this dependency is accurate from a database perspective (though it certainly is from a code one). I would be comfortable either with you listing the previously committed migration, or not listing anything.

Reply:
OK.  I'll remove the dependency, then.  Since we've agreed that the MiVideo data doesn't need to have a formal relationship with the course data, and the dependency on `job_run` doesn't count, because it's programmatic only, removing it is the best choice.